mne_features/feature_extraction.py

```diff
@@ -31,10 +31,15 @@ class FeatureFunctionTransformer(FunctionTransformer):
         If True, the array X will be checked before calling the function.
         If possible, a 2d Numpy array is returned. Otherwise, an exception
         will be raised. If False, the array X is not checked.
+
+    kw_args : dict or None (default: None)
+        If not None, dictionary of additional keyword arguments to pass to the
+        feature function.
     """
-    def __init__(self, func=None, validate=True):
+    def __init__(self, func=None, validate=True, kw_args=None):
```

Comment:
call kw_args params as in sklearn

Reply:
kw_args seems to be used the same way in https://github.com/scikit-learn/scikit-learn/blob/a24c8b464d094d2c468a16ea9f8bf8d42d949f84/sklearn/preprocessing/_function_transformer.py#L71. What am I missing ?